include/avif/avif.h

```diff
@@ -236,6 +236,11 @@ typedef struct avifPixelFormatInfo
     int chromaShiftY;
 } avifPixelFormatInfo;
 
+// Returns the avifPixelFormatInfo depending on the avifPixelFormat.
+// The chromaShiftX and chromaShiftY fields should be ignored when monochrome is AVIF_TRUE.
+// The chromaShiftX and chromaShiftY fields are set to 1 if the format is AVIF_PIXEL_FORMAT_YUV400 to allow
+// an AV1 implementation that only supports profile 0 to hardcode subsampling_x and subsampling_y to 1.
+// This function implements the second table on page 119 of the AV1 specification version 1.0.0 with Errata 1.
```

Comment:
Nit: Change these two sentences to the following (note the blank line and the Note):

```
//
// Note: This function implements the second table on page 119 of the AV1 specification version 1.0.0 with Errata 1.
// For monochrome 4:0:0, subsampling_x and subsampling are specified as 1 to allow
// an AV1 implementation that only supports profile 0 to hardcode subsampling_x and subsampling_y to 1.
```

Note: Another way to justify setting subsampling_x and subsampling_y to 1 for monochrome is that if we must add chroma planes (of neutral samples) to a monochrome image, then the 4:2:0 format uses the least amount of memory. For example, this is the reason behind the `avifImageCalcAOMFmt()` function in src/codec_aom.c.